packages/core/src/modules/credentials/CredentialsModule.ts

```diff
@@ -533,15 +542,15 @@ export class CredentialsModule<
       throw new AriesFrameworkError(`No connectionId found for credential record '${credentialRecord.id}'.`)
     }
     const connection = await this.connectionService.getById(credentialRecord.connectionId)
-
     const service = this.getService(credentialRecord.protocolVersion)
-    const problemReportMessage = service.createProblemReport({ message: options.message })
+
+    const problemReportMessage = options.code ? service.createProblemReport({ message: options.message, code: options.code }) : service.createProblemReport({ message: options.message })
```

Comment:
If you just send an undefined `code` to createProblemReport wouldn't save the same as the ternary operator? :thinking: